include/upipe-ts/upipe_ts_decaps.h

```diff
@@ -33,6 +33,19 @@ extern "C" {
 
 #define UPIPE_TS_DECAPS_SIGNATURE UBASE_FOURCC('t','s','d','c')
 
+enum upipe_ts_decaps_command {
+    UPIPE_TS_DECAPS_SENTINAL = UPIPE_CONTROL_LOCAL,
+
+    UPIPE_TS_DECAPS_GET_PACKETS_LOST, /* int sig, uint64_t * */
+};
+
+static inline int upipe_ts_decaps_get_packets_lost(struct upipe *upipe,
```

Comment:
Please add mkdoc comment as this is a public header. In particular you should state that the get modifies internal structures (of which I am not too fond of).